docs/content/about-iota/iota-wallet-dashboard/how-to/assets.mdx

```diff
@@ -0,0 +1,28 @@
+---
+description: How to manage NFTs and other assets in the IOTA Wallet Dashboard.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet-dashboard]
+---
+# Manage NFTs and Digital Assets
+
+You can access the Assets view by clicking on the `Assets` button on the left side menu.
+
+   ![Click Assets](/img/about-iota/iota-wallet-dashboard/how-to/assets/click-assets.png)
+
+You can use the buttons to filter by `Visual` and `Other` assets.
+
+   ![Assets View](/img/about-iota/iota-wallet-dashboard/how-to/assets/assets-view.png)
```

Comment:
Would it be possible to get another shot to be able to see the asset? atm is just a black dot